internal/pkg/cli/env_show.go

```diff
@@ -17,6 +17,8 @@ type showEnvVars struct {
 
 type showEnvOpts struct {
 	showEnvVars
+
+	storeSvc storeReader
```

Comment:
Since you're adding a new field to `showEnvOpts`, you will need to instantiate a store that implements the `storeReader` interface and set it as `o.storeSvc` in `newShowEnvOpts`. See: https://github.com/aws/amazon-ecs-cli-v2/blob/master/internal/pkg/cli/app_show.go#L45-L48